gnome-shell/src/gnome-shell-sass/_common.scss

```diff
@@ -633,7 +645,7 @@ StScrollBar {
   .level {
     height: 0.6em;
     -barlevel-height: 0.3em;
-    -barlevel-background-color: transparentize($osd_fg_color, 0.9);
+    -barlevel-background-color: transparentize($osd_fg_color, if($variant=='light', 0.2, 0.9));
```

Comment:
I think this is not needed for our theme, but I need to double check this

Reply:
Probably, our `$osd_fg_color` is the same for light and dark variant, but it doesn't hurt and reduces the diff

Reply:
It sadly hurts a little, I attached a screenshot in the comment below, the level bg becomes full white :/

Reply:
Yes, you're right. The `$variant` variable defaults to light, so instead of transparentize 0.9 we have 0.2 now, which is too bright

Reply:
Ok, I've reverted the change. I was quite unsure about it in fact :stuck_out_tongue: